docs/configuration/overview/tracing.rst

```diff
@@ -12,31 +12,30 @@ the only one supported.
 
   {
     "http": {
-      "sinks": []
+      "driver": "{...}"
     }
   }
 
 http
   *(optional, object)* Provides configuration for the HTTP tracer.
 
-sinks
-  *(optional, string)* Provides list of sinks traces are sent to.
+driver
+  *(optional, object)* Provides the driver that traces are sent to.
 
-Each sink can be configured separately. Currently only the `LightStep <http://lightstep.com/>`_ sink
-is supported.
+Currently only `LightStep <http://lightstep.com/>`_ is supported.
 
 .. code-block:: json
 
   {
-    "type": "lightstep",
```

Comment:
yes, i think access token needs to be moved under config, i agree. Will do that change this week.

it should be like:
driver: {
  "type": "X",
  "config" : {
    "DRIVER SPECIFIC CONFIG
  }
}